enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,767 @@
+|------|-----|------|------------|-------------|----------------------|
+| MCOA |  ✅  |   ✅  |      🚧     |             |                      |
+
+POC - https://github.com/ViaQ/logging-ocm-addon
+Demo - https://github.com/rhobs/multicluster-observability-addon/tree/main/demo
+DevPreview - https://github.com/rhobs/multicluster-observability-addon
+
+## Alternatives
+
+### Policy-Engine based workflow
+
+The RHACM policy engine is part of the RHACM governance tools to apply certain policies on workloads and infrastructure on the entire fleet. It serves well to a degree however for the MCOA use-case it falls short since MCOA will need to:
+- manage complext dependencies between multiple components, OpenShift Logging Operator installation (i.e Cluster-Logging-Operator) and the custom resources (i.e. ClusterLogForwarder) as well as authentication related artifacts (i.e. generating TLS certificates, Cloud Provider Managed Identities / ServiceAccounts, etc.).
+- build dynamic error reporting (e.g. operator installation/upgrade failures, CRD handling) and health reporting (e.g. running/ready progress of operators and operands).
+
+The spike for this alternative can be found here: https://gitlab.cee.redhat.com/openshift-logging/log-storage-toolbox/-/merge_requests/14
+
+### Integrate directly into present Multi-Cluster-Observability-Operator
+
+For the untrained eye the RHACM MCO operator looks like the natural place to add multi-cluster logging capabilities. It has a hybrid architecture being a Kubernetes operator (i.e managing the MultiClusterObservability CRD) and an addon (i.e. provisions observability agents/policies to clustersets). However, MCO nowadays has to many responsabilities which not only makes it hard to maintain but also hard to extend to further signals without a tremendous engineering effort.
+
+
+## Infrastructure Needed [optional]
+
+None
+
+[ocm-addon-framework]:https://github.com/open-cluster-management-io/addon-framework
+[ocp-cluster-logging-operator]:https://github.com/openshift/cluster-logging-operator
+[ocp-clusterlogforwarder-outputsecretspec]:https://github.com/openshift/cluster-logging-operator/blob/627b0c7f8c993f89250756d9601d1a632b024c94/apis/logging/v1/cluster_log_forwarder_types.go#L226-L265
+[ocp-clusterlogforward-outputtypespec]:https://github.com/openshift/cluster-logging-operator/blob/627b0c7f8c993f89250756d9601d1a632b024c94/apis/logging/v1/output_types.go#L21-L40
+[opentelemetry-collector-auth]:https://opentelemetry.io/docs/collector/configuration/#authentication
+[opentelemetry-operator]:https://github.com/open-telemetry/opentelemetry-operator
+[rhacm-multi-cluster-observability]:https://github.com/stolostron/multicluster-observability-operator
+
+## RANDOM  IDEAS 
+- What happens if we need to reference multiple static secrets?
+- Maybe we can use annotations for the authentication methods instead of having to create a seperate resouce for that
```

Comment:
What advantages do we currently have with the separate resource? I think this can be benefitial for users if it simplifies the workflow.